src/Cassandra/Requests/RequestExecution.cs

```diff
@@ -237,6 +238,16 @@ private void HandleSchemaChange(ResultResponse response)
                 {
                     var schemaAgreed = _session.Cluster.Metadata.WaitForSchemaAgreement(_connection);
                     result.Info.SetSchemaInAgreement(schemaAgreed);
+                    try
+                    {
+                        TaskHelper.WaitToComplete(
+                            _session.InternalCluster.GetControlConnection().HandleSchemaChangeEvent(schemaChange.SchemaChangeEventArgs, true),
```

Comment:
Yeah I noticed this while comparing the code between C# and node.js drivers. After I started seeing some tests failing because of this I decided to implement it right away. 

Without the debouncer the events were being processed fast enough that tests which created keyspaces on the spot weren't failling.